app/home.html

```diff
@@ -1,42 +0,0 @@
-<!DOCTYPE html>
```

Comment:
No worries.  I just saw an entire file get deleted and didn't see similar functionality throughout.  Not having the like/share buttons is fine by me

-----Original Message-----
From: "dani5447" <notifications@github.com>
Sent: ‎10/‎3/‎2015 11:58
To: "bbesmanoff/indoor-umbrellas" <indoor-umbrellas@noreply.github.com>
Cc: "Brian Besmanoff" <bbesmanoff@gmail.com>
Subject: Re: [indoor-umbrellas] Fixes #31 page redirects (#37)

In app/home.html:
> @@ -1,42 +0,0 @@
> -<!DOCTYPE html>
The like and share buttons aren't necessary for the project really. We could move them to the navbar if we really want? Idk what we want to do with that.
—
Reply to this email directly or view it on GitHub.